ensembl/htdocs/info/genome/compara/epo_anchors_info.html

```diff
@@ -6,52 +6,44 @@
 
 <body>
 
-<h2>The EPO pipeline is split into three parts</h2>
+<p>
+In the EPO pipeline, Enredo, Pecan and Ortheus are the tip of the iceberg.
+They require homology and synteny information that are derived from alignments of <em>anchor</em> sequences.
+This page explains what the anchors are, how they are generated and used.
+</p>
 
-<h3>1). Anchor generation</h3>
+<h3 id="anchorgeneration">Anchor generation</h3>
 
 <p>
 The anchors represent short regions of conservation, typically about 100 bases in length, from a phylogenetically representative subset of the species we wish to align.
-The anchor set is generated from pairwise alignments (LastZ-Net / BlastZ-Net) of a non-reference species to a reference species (human is the reference in our case), so all the species chosen to generate the anchors must have a pairwise alignment with the reference species. </p>
-  
-<p>We then use the alignment coordinates from the reference genome to find sets of overlapping sequences from the chosen <a href="analyses.html" class="bold">pairwise alignments</a>.</p>
+As such, anchors only have to be regenerated when the subset of species used is not representative any more.
+</p>
 
-<h3>2). Anchor mapping</h3>
 <p>
-The anchor set is mapped (currently we use exonerate) against all the genomes we wish to include in the final alignment. Overlapping anchor mappings are filtered so that any particular genomic location is associated with, at most, one anchor.  
+The anchor set is generated from pairwise alignments (LastZ-Net) of a non-reference species to a reference species, so all the species chosen to generate the anchors must have a pairwise alignment with the reference species.
+The pairwise alignments are stacked together based on their coordinates and the regions are realigned with <a href="https://github.com/benedictpaten/pecan">Pecan</a>.
```

Comment:
I think it would make sense to put the method links in the first line of this page, when they are first mentioned as well as here or instead of here